class_generator/scripts/tools.py

```diff
@@ -74,7 +73,7 @@ def generate_resource(kinds: List[str], yes: bool) -> None:
 @click.option(
     "--list-generated_file",
     is_flag=True,
-    help="List all generated files by `class_generator` under `ocp_resources` directory",
+    help="list all generated files by `class_generator` under `ocp_resources` directory",
```

Comment:
this can stay `List`

Reply:
Why? List needs to be imported and the recommended usage is `list`